### DISCUSSION TITLE
Does espanso support aarch64? How to compile

### DISCUSSION THREAD

**Discussion by fubingting** [espanso/espanso#1214] [Feature requests and Ideas]

**Comment by 4lch4** [comment]
Hey @fubingting, this might be what you're lookin' for:

https://espanso.org/install/linux/#installing-on-manjaro--arch

**Comment by jpmvferreira** [comment]
If you read the PKGBUILD you will see the line `arch=(x86_64)` in both espanso packages in the AUR which means that when installing on your system, for example using the AUR helper yay, you will get a warning because the PKGBUILD doesn't necessarily specify that `aarch64` is supported.

You can try to compile it regardless, i've seen packages available in the AUR which did not explicitly supported `aarch64` but turns out they were able to compile regardless. 

Give it a try, and if it works, notify the maintainer so that he's able to update the PKGBUILD accordingly, and also to stick around to help out and see if something breaks on that architecture, because he might not have a system with that architecture, remember this is all a community effort.

**Comment by federico-terzi** [comment]
@JPUnD Thank you for pointing this out! Espanso is currently undergoing a major rewrite, so we might try to address this problem as well in the process :)
